src/core/cardano/walletConnect/peerConnection.types.ts

```diff
@@ -0,0 +1,7 @@
+type ExperimentalAPIFunctions = {
+  getIdentifierId: any;
+  signData: any;
+  getOobi: any;
```

Comment:
These aren't `any` - they are functions as you defined them in the other file:
```
  getIdentifierId: () => Promise<string>;
  signDataWithIdentifier: (
    identifierId: string,
    payload: string
  ) => Promise<string>;
  generateOobi: (identifierId: string) => Promise<string>;
```

(our style guide is also to use `interface` unless we need specific `type` features)